compressed_depth_image_transport/include/compressed_depth_image_transport/rvl_codec.h

```diff
@@ -1,33 +1,39 @@
-#ifndef COMPRESSED_DEPTH_IMAGE_TRANSPORT_RVL_CODEC_H_
-#define COMPRESSED_DEPTH_IMAGE_TRANSPORT_RVL_CODEC_H_
+// Copyright 2012
```

Comment:
This doesn't seem right; the original header came is as part of commit a710d4abab2d203f2715203e2bce833d5b52a7cb in 2019, so I think that is what we should put here.